src/ucp/tag/eager_snd.c

```diff
@@ -55,13 +55,14 @@ static size_t ucp_tag_pack_eager_first_dt(void *dest, void *arg)
                                          sizeof(*hdr);
     hdr->super.super.tag = req->send.tag;
     hdr->total_len       = req->send.length;
+    hdr->tid             = req->send.message_id;
 
-    ucs_debug("pack eager_first paylen %zu", length);
+    ucs_debug("pack eager_first %"PRIu64" paylen %zu", hdr->tid, length);
```

Comment:
pls remove this (previously) invalid debug message

Reply:
not sure what is wrong is here... but removed